db/migrate/20230119181726_create_likes.rb

```diff
@@ -0,0 +1,12 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+
+      t.timestamps
+    end
+
+    add_reference :likes, :user, foreign_key: true
```

Comment:
- [ ] Same here you can add a reference for the `author` because the likes belong to the author
e.g
```ruby
t.references :author, foreign_key: { to_table: 'users' }
```